addon/components/base-focusable.js

```diff
@@ -10,6 +10,7 @@ export default Component.extend(EventsMixin, HasBlockMixin, {
   active: false,
   focused: false,
   hover: false,
+  focusable: true,
```

Comment:
It seems like maybe you could just set `focusOnlyOnKey` to `true` in `paper-switcth` to achieve the same effect. Or maybe I'm missing something? If you do need to keep `focusable`, maybe change the wordwrapping of the `if` in `focusIn()` to correspond to the semantics of the operator precedence as it is a touch misleading as wordwrapped.